scripts/lib/qa_crowbarsetup-help.sh

```diff
@@ -10,6 +10,8 @@ function qacrowbarsetup_help
         Option to set variable MTU size or select Jumbo Frames for Admin and Storage nodes. 1500 is used if not set.
     want_raidtype (default='raid1')
         The type of RAID to create.
+    want_database_sql_engine (default='postgresql')
```

Comment:
default='postgresql' is a bit misleading here, when we just use the crowbar default value (whatever that might be)